src/euphorie/client/docx/views.py

```diff
@@ -308,7 +308,10 @@ def _filename(self):
 
 
 class ActionPlanShortDocxView(ActionPlanDocxView):
-    _compiler = DocxCompilerShort
+    @property
+    @memoize
+    def _compiler(self):
+        return DocxCompilerShort
```

Comment:
What I mean is that leaving the code as it was, i.e.:
```suggestion
    _compiler = DocxCompilerShort
```
has to work as well.


```
>>> class A:
...     @property
...     def foo(self): return 1
... 
>>> class B:
...     foo = 2
... 
>>> A().foo
1
>>> B().foo
2
```